SyntaxFunctionsStatementsExercise/07-cooking-by-numbers.js

```diff
@@ -0,0 +1,35 @@
+function solve([num, ...operations]) {
+    const posibleOperations = {
+        'chop': chop,
+        'dice': dice,
+        'spice': spice,
+        'bake': bake,
+        'fillet': fillet,
+    };
+
+    return operations.map(generateFunction).join('\n');
+
+    function generateFunction(operation) {
+        return num = posibleOperations[operation](parseInt(num));
```

Comment:
Why return num= ? Can’t you just return:
```suggestion
        return posibleOperations[operation](parseInt(num));
```